mugoh/qstioner
# #163391872 Register and Log in User using Form Data

#### What does this PR do?
Allow a user to register and sign in to the application.

#### Description of Task to be completed?
- A user can enter their details on the log in form an create an account.
- A user can pass their credentials on the login form and access their account.

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkout ft-register-user-163391872
```
4. Open the `sign-up.html` file
``` shell
$ browser-name sign-up.html
```
or
Simply  load up on your preferred browser

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163391872](https://www.pivotaltracker.com/story/show/163391872)
